README.md

```diff
@@ -4,6 +4,7 @@
 
 - [Introduction](#introduction)
 - [Getting Started](#getting-started)
+  - [Note](#note)
```

Comment:
I would either omit this since “Note” takes space and attention without conveying anything—like a road sign that says “sign ahead”—or relabel it to something like “v2.5.0 Changes”.

I’m suggesting omission because it feels like change notes should be part of the changelog or release notes, and because the user is prompted to delete the orphaned global file and there’s presumably no other harm to leaving it around.

```suggestion
```